projects/storefrontstyles/scss/components/product/details/_product-summary.scss

```diff
@@ -17,6 +17,9 @@
     margin: 0 0 11px 0;
   }
 
+  p.summary p {
+    margin-bottom: 0;
```

Comment:
hmm... not sure about that, but it seems we are essentially targetting existing element (already present) with different styles, what do you think?

Reply:
will remove that as it's better to make such adjustments using a theme